packages/types/lib/appium-config.ts

```diff
@@ -49,10 +49,23 @@ export type LocalTimezoneConfig = boolean;
  * Also send log output to this file
  */
 export type LogConfig = string;
+/**
+ * Log filtering rule
+ */
+export type LogFilter = {
```

Comment:
I just gotta say that [json-schema-to-typescript](https://npm.im/json-schema-to-typescript) is pretty awesome.  thanks @bcherny